examples/rules_cc/.bazelrc

```diff
@@ -0,0 +1,5 @@
+# Copyright 2023 Uber Technologies, Inc.
+# Licensed under the MIT License
+
+build --incompatible_enable_cc_toolchain_resolution
+build --action_env BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
```

Comment:
let's add the filesystem restrictions somewhere.

e.g.

```
config:restricted --experimental_use_hermetic_linux_sandbox
config:restricted --sandbox_writable_path="/tmp/..."
config:restricted --sandbox_add_mount_pair=/proc
```

Reply:
See previous comment